python/cudf_polars/cudf_polars/experimental/distinct.py

```diff
@@ -46,8 +50,8 @@ def lower_distinct(
         associated partitioning information.
     config_options
         GPUEngine configuration options.
-    cardinality
-        Cardinality factor to use for algorithm selection.
+    unique_fraction
+        Fractional unique count to use for algorithm selection.
```

Comment:
Nitpick, but since CI will need to rerun anway:

```suggestion
        Fraction of unique values to total values. Used for algorithm selection.
        A value of `1.0` means the column is unique.
```